Meadowlark-js/tests/e2e/functions/Resources.ts

```diff
@@ -19,8 +20,12 @@ export async function createResource({
     .post(`/v3.3b/ed-fi/${endpoint}`)
     .auth(await getAccessToken(credentials), { type: 'bearer' })
     .send(body)
-    .expect(201)
     .then((response) => {
+      if (response.body) {
+        console.error(response.body);
```

Comment:
This seems reasonable in test code. Should we customize your slide not rules for these tests to stop flagging console access?